### DISCUSSION TITLE
It seems that some unusual accounts starred/forked/followed my repository

### DISCUSSION THREAD

**Discussion by fzyzcjy** [community/community#84177] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Thanks for the GitHub ecosystem! However, when viewing my homepage, I noticed some unusual things (a bit similar to last year), and thus digged a bit and here is the report.

Firstly, screenshots of my feed:

<details>

![image](https://github.com/community/community/assets/5236035/22616c2d-2b84-4683-aaac-b5677decce0b)
![image](https://github.com/community/community/assets/5236035/09075b42-3a83-4bbf-9fa1-bd4923190f9e)
![image](https://github.com/community/community/assets/5236035/1f6466d0-d835-4824-bebf-3d82329a9614)

</details>

Their accounts:

* https://github.com/LovellCulbreath
* https://github.com/DashiellMckelphin
* https://github.com/ZandraBlon

Screenshots of the accounts:

<details>

![image](https://github.com/community/community/assets/5236035/6f0d6557-d8d7-4070-8de1-34a0487e857d)
![image](https://github.com/community/community/assets/5236035/925876dc-43cc-418b-9880-b5c45f830104)
![image](https://github.com/community/community/assets/5236035/f26e2c6c-210c-41cc-bb6d-cbd6da3f2934)

Screenshot of the tictactoe:

![image](https://github.com/community/community/assets/5236035/35f46cbb-f20f-43e0-bccd-3f81173db58b)
![image](https://github.com/community/community/assets/5236035/7ca9f91a-deae-4840-a97b-868295a83d03)
![image](https://github.com/community/community/assets/5236035/bd858eed-ebac-4332-99e2-eb00247c7383)

Screenshot of the repo:

![image](https://github.com/community/community/assets/5236035/a042a6f8-9179-4bca-a25c-8357d2176926)
![image](https://github.com/community/community/assets/5236035/585f6127-cce5-417b-87f4-d83cc3ca7876)
![image](https://github.com/community/community/assets/5236035/1e4f40b3-8213-4bef-a9a2-865db784047e)

</details>

Things that I feel a bit weird:

* All joined last week
* All have a website link that is the same name as the company specified.
* Moreover, all websites cannot be opened.
* All creates a `tictactoe` repository, with the same content, and same commit message "My First commit"
* All creates another repository with three-word names, with the same README style, same two-commits.

Since this report is similar to [the one last year](https://github.com/orgs/community/discussions/53643), I will copy-paste (and slightly modify) paragraphs that applies to here:

I want to believe that they are just new users of GitHub, and it is just randomness that brings this scenario, and if I wrongly suspect them while they are real user, I do apologize for that! 

However, the reason why I post this thread is that, I have heard that some people are creating fake accounts on GitHub and pay money to get fake stars, or use any approach to get the stars that should not have belong to them. I HATE that very much, which makes the stars on GitHub meaningless, if such behavior become a standard. I do value the stars/watches/forks I get, because I think that can be a measure of how many people are using/loving my library and I am proud that my tooling can help many people.

Therefore, when I seeing the unusual accounts above, I suspect it is some account that will be used to such pay-to-get-fake-stars behavior above. Thus, (1) I do hope such pay-to-get-stars accounts (if they are really fake and pay-to-get-stars account, which offenses GitHub policy) can be banned (2) I do hope no such fake accounts give stars/forks to my own repository,.

P.S. I do hope to create a github ticket, or use the "report abuse" button, but as mentioned in a previous discussion, there does not seem to exist a proper category for me to choose :(

(p.s. I am using the open-source hosted-on-github Refined GitHub plugin so the screenshots may be a little bit different from what you view it, but the things are mainly the same)

P.S. There are also other new accounts, etc, but does not share the pattern mentioned above, thus surely I should not regard every new user as non-real accounts. Therefore, if GitHub have time, I would appreciate it if GitHub could also check those accounts (just look at people who forked/starred flutter_rust_bridge).

### EDIT: There seems to be one more account

* https://github.com/Stepanie6

<details>

![image](https://github.com/community/community/assets/5236035/140abf7f-396d-48a1-9405-106ea5f693e9)
![image](https://github.com/community/community/assets/5236035/48e3ea26-33ac-4720-8222-a90220b7f082)
![image](https://github.com/community/community/assets/5236035/f7a60944-80db-42bf-8141-d2ddf2d3d557)

</details>

### EDIT 2

I checked my follower list just now. It seems that some are such similar unusual accounts.

<details>

My follower list:

![image](https://github.com/community/community/assets/5236035/e7cb4018-e6c4-4898-b0f6-d20b50146741)
![image](https://github.com/community/community/assets/5236035/247cf936-3871-4ef4-970c-8d3d76d4029f)
![image](https://github.com/community/community/assets/5236035/bbf1ad64-0da5-425d-b483-d751d5f982cd)
![image](https://github.com/community/community/assets/5236035/b7663c99-b282-4c09-8fa1-594e1d824d1b)

</details>

Unusual accounts similar to the pattern above are:

* https://github.com/DerrickVd
* https://github.com/ColtonPa
* https://github.com/OmaHarell
* https://github.com/MarkettaCl
* https://github.com/VerniceWi
* https://github.com/FaithCas
* https://github.com/Katheryn6
* https://github.com/ArvelVass
* https://github.com/Stepanie6
* https://github.com/ConwayPoister
* https://github.com/GeraldineOttey
* https://github.com/BernieceStransky
* https://github.com/KinaShuman
* https://github.com/JoanieGlaab
* https://github.com/JuleneLeppink
* https://github.com/MaudeGaver
* https://github.com/GoldaKihn
* https://github.com/LovellCulbreath
* https://github.com/ChristobelRona
* https://github.com/GaleKapuscinski
* https://github.com/DashiellMckelphin

with screenshots:

<details>

![image](https://github.com/community/community/assets/5236035/c210a785-673f-43b2-bba1-a0bc7c0a36d4)
![image](https://github.com/community/community/assets/5236035/70d23a6b-3c13-4bd7-a380-6ea5f21b6217)
![image](https://github.com/community/community/assets/5236035/6d157042-7e21-4417-a843-1bfa60d0763f)
![image](https://github.com/community/community/assets/5236035/c8b111e5-f5bf-4d50-aef7-706fa8f45a42)
![image](https://github.com/community/community/assets/5236035/22da2994-96a7-4deb-b38c-b2825567f863)
![image](https://github.com/community/community/assets/5236035/1ee29a10-dbbb-41f9-a074-7c56b3237f80)
![image](https://github.com/community/community/assets/5236035/7c88869f-147f-4d90-aae8-a347848ac59a)
![image](https://github.com/community/community/assets/5236035/615d518b-f226-4443-b838-d7411251f8a3)
![image](https://github.com/community/community/assets/5236035/161770c5-f0fd-4c5c-b780-b6e15e221ad3)
![image](https://github.com/community/community/assets/5236035/45c7ea4e-b1cc-4c0d-a6d0-0430e3001622)
![image](https://github.com/community/community/assets/5236035/2056287d-369b-420f-be33-6a6a728ff986)
![image](https://github.com/community/community/assets/5236035/2a526d54-00d3-4634-94a5-e9b8fae48530)
![image](https://github.com/community/community/assets/5236035/1fbdde93-a606-42c8-92ab-37640efd1cfb)
![image](https://github.com/community/community/assets/5236035/594036a7-be46-4400-a2ac-bea0e7c9c67e)
![image](https://github.com/community/community/assets/5236035/702b9615-62dc-413a-a12c-33a035c851b4)
![image](https://github.com/community/community/assets/5236035/06fc4ef8-85e0-4e17-a53f-12c9ae0afa99)
![image](https://github.com/community/community/assets/5236035/754b812d-a9d5-4616-9a11-0631a29a26be)
![image](https://github.com/community/community/assets/5236035/e55f8174-32b3-4974-b76c-ba150729683c)
![image](https://github.com/community/community/assets/5236035/13607702-7b99-4a04-8f22-a0573af0399c)
![image](https://github.com/community/community/assets/5236035/76cdbd82-36db-40c7-9b69-9e23ce7b3477)
![image](https://github.com/community/community/assets/5236035/f8874b51-8381-4d34-999a-f91c3a052f39)

</details>

### EDIT 3: I `block`ed these accounts now.

But I do hope GitHub can give a conclusion, and avoid scenarios of bad accounts!

Full list (gathered from above):

* https://github.com/DerrickVd
* https://github.com/ColtonPa
* https://github.com/OmaHarell
* https://github.com/MarkettaCl
* https://github.com/VerniceWi
* https://github.com/FaithCas
* https://github.com/Katheryn6
* https://github.com/ArvelVass
* https://github.com/Stepanie6
* https://github.com/ConwayPoister
* https://github.com/GeraldineOttey
* https://github.com/BernieceStransky
* https://github.com/KinaShuman
* https://github.com/JoanieGlaab
* https://github.com/JuleneLeppink
* https://github.com/MaudeGaver
* https://github.com/GoldaKihn
* https://github.com/LovellCulbreath
* https://github.com/ChristobelRona
* https://github.com/GaleKapuscinski
* https://github.com/DashiellMckelphin
* https://github.com/ZandraBlon

Screenshots

<details>

before:

![image](https://github.com/community/community/assets/5236035/6dbe0a3d-0062-46ea-930c-b8d46cdc5238)

after:
(this is slightly outdated, I missed out some accounts. directly see my github page for latest)

![image](https://github.com/community/community/assets/5236035/3d2f9c65-a69a-4b94-bd54-b1719572a8a4)
![image](https://github.com/community/community/assets/5236035/3a38b1aa-bc96-4015-ac34-a3d208bbd4fa)
![image](https://github.com/community/community/assets/5236035/8282586e-94ac-4ba9-8907-a199d4acfdd2)

</details>

### EDIT 4: More account to block and report

* https://github.com/ShaunteM 
* https://github.com/JacquelinBazin
* https://github.com/TawnyLermon
* https://github.com/NicZiff
* https://github.com/RosauraSo
* https://github.com/PollieMcdonough
* https://github.com/SienaKezele

<details>

![image](https://github.com/community/community/assets/5236035/da2834af-af58-41cd-8944-abf2a2f59547)
![image](https://github.com/community/community/assets/5236035/7e6b216d-61b5-4e80-97d8-5d3bd0f20c70)

![image](https://github.com/community/community/assets/5236035/606d7ed7-b228-49d3-ab50-3f29e1a52fa2)
![image](https://github.com/community/community/assets/5236035/d6e48576-5d6c-4e9f-8e92-2268e753793e)
![image](https://github.com/community/community/assets/5236035/bc3cda5e-b8a8-429b-9c9d-f0e1a126add3)

![image](https://github.com/community/community/assets/5236035/4d7ba8f6-1e42-4868-b7da-1398cb31cfa8)
![image](https://github.com/community/community/assets/5236035/40e9e1dc-1416-4851-8953-afb987278ae9)
![image](https://github.com/community/community/assets/5236035/70828039-399a-4bba-a226-1c4f369a7826)
![image](https://github.com/community/community/assets/5236035/3c6fb317-1691-47ed-966c-710a4efdaff3)
![image](https://github.com/community/community/assets/5236035/09166bca-e4b2-45c2-93a7-d5e8ae5e960e)

</details>

### EDIT 5: More

* https://github.com/LizabethOlberding
* https://github.com/JohnnieEnock
* https://github.com/IndianaDelosreyes
* https://github.com/GwendolineQueller
* https://github.com/MelissaCaramanica
* https://github.com/MosesKorineck
* https://github.com/MargeBrickett
* https://github.com/HaideeDoerle
* https://github.com/ElmiraBredeweg
* https://github.com/RosettaWittler

<details>

![image](https://github.com/community/community/assets/5236035/8207c3c6-b9f8-4045-a786-a83c2b194fc7)

![image](https://github.com/community/community/assets/5236035/7d5594ee-bc5a-4aa0-a4c4-7ef31c68863f)
![image](https://github.com/community/community/assets/5236035/0ca6b191-9ae2-4133-8f1e-d01a82952d37)
![image](https://github.com/community/community/assets/5236035/5765c6fe-bd1b-4eab-813c-057ffbd45b96)
![image](https://github.com/community/community/assets/5236035/c019907e-a381-4ff4-9845-4172e4fcf7a4)
![image](https://github.com/community/community/assets/5236035/6028ac05-fdec-42c5-a8cf-9aad0cb28f9c)
![image](https://github.com/community/community/assets/5236035/9ff24014-ee1f-40cf-8d4a-891b334f90c5)
![image](https://github.com/community/community/assets/5236035/2f6afada-ee92-4149-84ac-ef1042516636)
![image](https://github.com/community/community/assets/5236035/95b3ca89-3516-4af1-85ab-3fc1ba2fac08)
![image](https://github.com/community/community/assets/5236035/d70384bc-d2cb-49f1-baf9-2c908591e1ca)
![image](https://github.com/community/community/assets/5236035/6d020bae-4952-418c-ad21-5c9ea0ed7f01)

</details>

### EDIT 6: More

* https://github.com/RobynThurgood
* https://github.com/JaymieWojtecki
* https://github.com/CicelyReale
* https://github.com/FloridaBrackemyre
* https://github.com/CharisseKypuros
* https://github.com/JenaCsaszar
* https://github.com/SachaSellars

<details>

![image](https://github.com/community/community/assets/5236035/8c6b0c6e-3a38-475b-9189-de925c50f12c)

![image](https://github.com/community/community/assets/5236035/a9fd336a-7a75-47cd-8eb6-fc81ed2f3332)
![image](https://github.com/community/community/assets/5236035/38abee15-60d4-4d62-be1b-f126a5ee1c49)
![image](https://github.com/community/community/assets/5236035/60b51ba7-e837-4c79-90f2-683b2f8a85c9)
![image](https://github.com/community/community/assets/5236035/7ee51ba5-31bb-4201-be03-d8a494066d7a)
![image](https://github.com/community/community/assets/5236035/9516896c-7527-430c-9943-896a736ef4e2)
![image](https://github.com/community/community/assets/5236035/2d1dbbd1-3c56-44cb-9015-2eb2e655fa52)
![image](https://github.com/community/community/assets/5236035/28e30018-e89e-417a-b85d-d31da2db3f2c)

</details>

### EDIT 7: More

* https://github.com/AmarillaGangwish
* https://github.com/KatheZabbo
* https://github.com/HarmonieStrawn
* https://github.com/CobyMcnamee

<details>

![image](https://github.com/community/community/assets/5236035/24109a40-f18b-4e44-987a-e80f54060c1a)

![image](https://github.com/community/community/assets/5236035/6bb14538-e19b-4b28-80ea-18e09f5b6f5d)
![image](https://github.com/community/community/assets/5236035/0661bc12-0912-4993-a6b0-abc545d7724d)
![image](https://github.com/community/community/assets/5236035/aea6b94c-cd6b-4942-ac8a-9a58f002a957)
![image](https://github.com/community/community/assets/5236035/8529e603-4dc5-4ce9-bba3-3b354d657865)

</details>

**Comment by twpayne** [comment]
I'm also getting spammed with fake followers in my feed. See screenshot below.

I don't think GitHub cares about spam. For them, it's just more users, which looks good in their metrics.

![fake-accounts](https://github.com/community/community/assets/6942/7565899f-d292-4f43-b6d3-c886d364c571)
